css/style.css

```diff
@@ -64,55 +68,57 @@ h2 {
 .main-text {
   width: 100%;
   padding: 20px;
-  padding-right: 100px;
+  height: 100px;
   text-align: justify;
   font-family: 'Raleway', 'Helvetica';
 }
 
 .wrapper {
-  max-width: 960px;
+  max-width: 1150px;
   margin: 0px auto;
+  /*background-color: #e3f2f9;*/
 }
 
 
-/*Media accessibility stylings*/
+input[type=text] {
+  font-family: 'Raleway', 'Helvetica';
+  padding: 5px;
+  margin-top: 20px;
+}
 
+input[type=email] {
```

Comment:
Since you have the same styling on both these, you could use the multiple selector syntax and do this:

```
input[type=text], input[type=email] {
 Your css goes here
}
```